### DISCUSSION TITLE
Converting Float32 to Float64

### DISCUSSION THREAD

**Discussion by jblindsay** [modular/modular#2633] [Q&A] [answered]
How do you convert between numeric data types in Mojo? For example, how would you convert a Float32 to a Float64?

**Comment by mikowals** [comment, marked as answer]
```
var x = Float32(1)
var y = x.cast[DType.float64]()
print(y.type)
```
See [docs of cast](https://docs.modular.com/mojo/stdlib/builtin/simd/SIMD#cast).

**Reply by jblindsay** [reply]
That's exactly it @mikowals thanks. Wow, that was not so obvious to me. I would have expected something more like:

```python
var x = Float32(1)
var y = Float64(x)
```

But that doesn't seem to work well.
